test/table.js

```diff
@@ -174,6 +175,20 @@ describe('Table', () => {
       assert.deepEqual(rows.length, 231)
     })
 
+    it('should autodetect CSV separated by semicolons', async () => {
+      const data = `city;location;population
+London;51.50,-0.11;8.788
+Paris;48.85,2.30;2.244
+`
+      const stream = Readable()
+      const table = await Table.load(stream)
+      stream._read = () => {}
+      stream.push(data)
+      stream.push(null)
+
+      const rows = await table.read()
+      assert.equal(rows.length, 2)
```

Comment:
This assertion doesn't seem really testing separators.

Reply:
The intent was to test that the file was loaded correctly, but I think you're saying that it could be that it was loaded as a single column (for example). I'll change it to test some value :+1: